engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/ExecCompilerTest.java

```diff
@@ -17,25 +17,27 @@
 import org.enso.common.MethodNames.Module;
 import org.enso.common.RuntimeOptions;
 import org.enso.compiler.core.ir.expression.errors.Conversion.DeclaredAsPrivate$;
+import org.enso.test.utils.ContextRule;
 import org.graalvm.polyglot.Context;
 import org.graalvm.polyglot.PolyglotException;
 import org.graalvm.polyglot.Source;
 import org.graalvm.polyglot.io.IOAccess;
 import org.hamcrest.core.AllOf;
 import org.junit.After;
 import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Ignore;
 import org.junit.Test;
 
 public class ExecCompilerTest {
-  private static Context ctx;
-  private static ByteArrayOutputStream out;
+  @ClassRule
+  public static final ContextRule ctxRule =
+      ContextRule.createCustom(ExecCompilerTest::initEnsoContext);
 
-  @BeforeClass
-  public static void initEnsoContext() {
-    out = new ByteArrayOutputStream();
-    ctx =
+  private static final ByteArrayOutputStream out = new ByteArrayOutputStream();
```

Comment:
Remove this static `out` field. It's ugly, and it is a source of potential memory leaks.

Reply:
Done in bdfaa6bfaaa3052d7baac7f46074f530ff2ed974